src/google/protobuf/compiler/objectivec/objectivec_helpers.cc

```diff
@@ -621,6 +621,33 @@ ObjectiveCType GetObjectiveCType(FieldDescriptor::Type field_type) {
   return OBJECTIVECTYPE_INT32;
 }
 
+string GetZeroEnumNameForFlagType(const FlagType flag_type) {
```

Comment:
Move these two up into the anon namespace (namespace { ... }), so they are private to this file and don't become global symbols.